modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -88,5 +65,3 @@ The `PolicyGenTemplate` CRs required to deploy PTP fast events are described bel
 . Merge any other required changes and files with your custom site repository.
```

Comment:
I believe AmqInstance.yaml is also needed. @aneeshkp could you please provide an example to doc? We can compare it with ours after ipv6 AMQ deployment testing is completed.

Reply:
Like: https://gitlab.cee.redhat.com/ran/lab-ztp/-/blob/westford-lab/configuration/cnfde7/group-cnfde7.yaml#L58 ?

Reply:
@yliu127 Created a PR to address this: https://github.com/amolnar-rh/openshift-docs/pull/6/files#diff-fbbd32dc748eabd444146432bb881797090c3bc068db2f8aa13d544a1cfd0117R36

Reply:
Confirmed with @aneeshkp. Updating...

Reply:
Updated.